### PR TITLE
Login signup

### PR DESCRIPTION
**Pull request by DivyanshSareen**
added login and signup pages(no auth)

### CODE REVIEW

File: src/components/Cards/PlaylistFormCard.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const PlaylistFormCard = ({ video }) => {
+  console.log(video);
```
Comment (sudipkundu999) [inline src/components/Cards/PlaylistFormCard.jsx]: remove console logs

**Review by sudipkundu999 — APPROVED** [review]
Great work.
- Nice structured code
- Good abstraction of all providers

Suggestion: add a guest login feature

File: src/components/Forms/LoginForm.jsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import { Link } from "react-router-dom";
+import { useAuth } from "../../context/auth-context";
+
+const LoginForm = () => {
+  const { authState, authDispatch } = useAuth();
+  const { email, password, remember_me } = authState.loginForm;
```
Comment (humzahasan) [inline src/components/Forms/LoginForm.jsx:6]: Very good approach!

File: src/components/Forms/LoginForm.jsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      </div>
+      <div className='btn btn-login center-text'>Log In</div>
+      <Link to='/signup'>
+        <div className='paragraph1 center-text'>
+          Create New Account <i className='fa-solid fa-chevron-right'></i>
+        </div>
+      </Link>
+    </form>
+  );
+};
```
Comment (humzahasan) [inline src/components/Forms/LoginForm.jsx:76]: You are firing 3 different actions, you can have it as one common action for login too right ? Any specific reason ?

**Review by humzahasan — APPROVED** [review]
Code changes looks good! 

Please do get back on the question of firing multiple actions instead of one. I have seen you used this at multiple places. 

Good to merge!

File: src/components/Forms/LoginForm.jsx (hunk 3 above)
Comment (DivyanshSareen) [reply src/components/Forms/LoginForm.jsx:76]: Thank you for the review. I thought of using useState to monitor inputs. Then I remembered that it uses useReducer under the hood and it made sense to keep these values under auth-context and reducer. Do let me know your take on this.

**Pull request merged**